docs/CACHING.md

```diff
@@ -0,0 +1,353 @@
+# Caching Configuration of Mountpoint for Amazon S3 CSI Driver
+
+Mountpoint supports caching file system metadata and object content to reduce cost and improve performance for repeated reads to the same file. The CSI Driver allows you to configure caching of Mountpoint in your PersistentVolume (PV) definition. See [Mountpoint's caching configuration]((https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#caching-configuration)) for more details about caching.
```

Comment:
np: `Mountpoint` - not clear what you mean by this. How about `[Mountpoint for Amazon S3](https://github.com/awslabs/mountpoint-s3)`?